terraform/aws/data/architecture/arm.yaml

```diff
@@ -0,0 +1,8 @@
+---
+image::architecture::arm: 'arm64'
+# instance metadata
+instance::arm::micro: 't4g.micro'
```

Comment:
validate if we can do hashes and leverage in TF script.

Reply:
tried this by putting these keys into hash but that didn't worked as expected. so keeping as it is.